src/ingestor/typedData/index.ts

```diff
@@ -36,20 +36,33 @@ export default async function ingestor(body) {
     return Promise.reject('wrong domain');
 
   const hash = sha256(JSON.stringify(types));
-
   if (!Object.keys(hashTypes).includes(hash))
     return Promise.reject('wrong types');
   let type = hashTypes[hash];
 
   if (
-    !['settings', 'alias'].includes(type) &&
```

Comment:
How is it better to do it this way?

Reply:
Its same just lesser conginitive load to understand. 
![image](https://user-images.githubusercontent.com/91108642/164127279-df0f2589-31d3-4864-903b-112cdac48741.png)

Reply:
But then you are adding a much bigger list, I'd rather map the action that doesn't require space than the opposite.